### PR TITLE
add Service examples

### PR DESCRIPTION
**Pull request by larsl-net**
Create a collection of sample configurations to monitor different services

### CODE REVIEW

File: content/en/docs/Examples/services.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+
+To get the version from a Enterprise Edition change the `ce` to `ee` in the `url_commands` `regex`.
+```yaml
+service:
+  gitlab-org/gitlab:
+    type: url
+    url: https://gitlab.com/api/v4/projects/278964/repository/tags
+    url_commands:
+      - type: regex_submatch
+        regex: \"name\":\"v([0-9.]+\-ce)"
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:55]: Missing a `\` on the last `"`?

File: content/en/docs/Examples/services.md [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+```yaml
+services:
+  healthchecks/healthchecks:
+    type: github
+    url: healthchecks/healthchecks
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/mailcow/mailcow-dockerized/releases/tag/{{ version }}
+    icon: https://raw.githubusercontent.com/mailcow/mailcow-dockerized/master/data/web/img/cow_mailcow.svg
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:204]: Unrelated icon and web_url

**Comment by JosephKav** [comment]
Good work so far!
Yeah, it's still a WIP, I just thought I may as well point them out them out in-case you don't look back at those ones :)

File: content/en/docs/Examples/services.md (hunk 1 above)
Comment (larsl-net) [reply content/en/docs/Examples/services.md:55]: Yes that is missing thank you

File: content/en/docs/Examples/services.md (hunk 2 above)
Comment (larsl-net) [reply content/en/docs/Examples/services.md]: web_url is changed to the right project.

The icon is a bit more difficult as there is no icon in the repo itself, only a placeholder.

File: content/en/docs/Examples/services.md [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+```yaml
+services:
+  prometheus/prometheus:
+    type: github
+    url: prometheus/prometheus
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/prometheus/prometheus/blob/main/CHANGELOG.md
+    regex_content: prometheus-{{ version }}\.linux-amd64
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:346]: Not sure whether you want `deployed_version` everywhere we know a url, but I think we should? Here's the Prometheus one
```
    deployed_version:
      url: https://prometheus.example.io/api/v1/status/buildinfo
      json: version
```
(same for Thanos, just thanos in the url)

File: content/en/docs/Examples/services.md [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+```yaml
+service:
+  prometheus/alertmanager:
+    type: github
+    url: prometheus/alertmanager
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/prometheus/alertmanager/blob/main/CHANGELOG.md
+    regex_content: alertmanager-{{ version }}\.linux-amd64.tar.gz
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:332]:
```yaml
    deployed_version:
      url: https://alertmanager.example.io/api/v1/status
      json: version
```

File: content/en/docs/Examples/services.md (hunk 3 above)
Comment (larsl-net) [reply content/en/docs/Examples/services.md:346]: Yes would enter the `deployed_version` everywhere we have a URL for it.

File: content/en/docs/Examples/services.md [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+    regex_content: gitea-{{ version }}-linux-amd64
+    regex_version: ^[0-9.]+[0-9]$
+    icon: https://raw.githubusercontent.com/go-gitea/gitea/main/public/img/logo.png
+```
+
+## go-vikunja/api
+Source: https://github.com/go-vikunja/api
+```yaml
+service:
+  go-vikunja/api:
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:103]:
```yaml
    deployed_version:
      url: https://vikunja.example.io/api/v1/info
      json: version
      regex: v?([0-9.]+)
```

File: content/en/docs/Examples/services.md [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+    web_url: https://github.com/gohugoio/hugo/releases/v{{ version }}
+    regex_content: hugo_{{ version }}_Linux-64bit\.deb
+    icon: https://raw.githubusercontent.com/gohugoio/hugo/master/docs/static/img/hugo.png
+```
+
+## gotify/server
+Source: https://github.com/gotify/server
+```yaml
+service:
+  gotify/server:
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:169]:
```yaml
    deployed_version:
      url: https://gotify.example.io/version
      json: version
```

File: content/en/docs/Examples/services.md [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+    deployed_version:
+      url: https://hedgedoc.example.io/config
+      regex: window.version = '([0-9.]+)-[0-9a-z]+'
+```
+
+## home-assistant/core
+Source: https://github.com/home-assistant/core
+```yaml
+service:
+  home-assistant/core:
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:249]:
```yaml
    deployed_version:
      url: https://home-assistant.example.io/api/config
      headers:
        - key: Authorization
          value: Bearer <API_TOKEN>
```

API_TOKEN from going to `/profile` and creating a 'Long-Lived Access Token'

File: content/en/docs/Examples/services.md [hunk 8]
```diff
@@ -0,0 +1,569 @@
...
+    type: github
+    url: prometheus/alertmanager
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/prometheus/alertmanager/blob/main/CHANGELOG.md
+    regex_content: alertmanager-{{ version }}\.linux-amd64.tar.gz
+    deployed_version:
+      url: https://alertmanager.example.io/api/v1/status
+      json: version
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:462]: Could you update this to `data.versionInfo.version` (AlertManager), and both Prometheus/Thanos to `data.version` please?

Comment (larsl-net) [reply content/en/docs/Examples/services.md]: Done

**Pull request merged**